src/duration.rs

```diff
@@ -232,6 +232,20 @@ impl Duration {
         }
     }
 
+    /// Returns the number of microseconds such that
+    /// `subsec_micros() + num_seconds() * MICROS_PER_SEC` is the total number of
+    /// microseconds in the duration.
+    pub const fn subsec_micros(&self) -> i32 {
+        self.subsec_nanos() / NANOS_PER_MICRO
+    }
+
+    /// Returns the number of milliseconds such that
+    /// `subsec_millis() + num_seconds() * MILLIS_PER_SEC` is the total number of
```

Comment:
I like this specificity.

@pitdicker should `MILLIS_PER_SEC` be made `pub`?  If a user wants to follow this docstring implied suggestion (compute `subsec_millis() + num_seconds() * MILLIS_PER_SEC`) then they'll want to access `MILLIS_PER_SEC`.  This docstring is taken from `subsec_nanos` which has the same implied suggestion using `NANOS_PER_SEC`, so `NANOS_PER_SEC` should probably also be `pub`.

Stretch goal would be to make all the constants in that clump of constants above into `pub`.

Reply:
I think it makes sense to make these constants part of the public API.

Reply:
A made a change like that to the docstrings. I'm wondering if those constants should stay in the `duration` module. They might make more sense if they were put in the `lib.rs`. I can't imagine anyone looking for these constants intuiting that they'd be in `duration`.

Reply:
Makes sense to me!

Reply:
Just made that change, but I do have one more question. Now that these are public, I find it a bit strange that some of them are `i32` and some are `i64`. I think this makes sense for what we're using them for, and if it's useful for us to have them that way, then it's probably useful for someone else too, but I wanted to bring it in case someone thinks we should make it more consistent.

Reply:
I believe the goal for using `i32` is to lessen resource "footprint" chrono objects.